src/EIPTestsFiller/StateTests/stEOF/stEIP3540/CREATE2_EOF1Filler.yml

```diff
@@ -43,42 +42,40 @@ CREATE2_EOF1:
       #- ':yul { mstore8(0, 0xef) return(0, 32) }'
       - ':label deploying_0xef00..00 :raw 0x60ef60005360206000f3'
 
-      # Valid EOF1 initcode and code combinations (both with and withoug data section)
+      # Valid EOF1 initcode and code combinations (both with and without data section)
 
       # Legacy initcode => EOF code
       # deployed code without data section
       # deployed code section: PUSH1(0) POP STOP
-      - ':label deploying_valid_eof1 :yul { mstore(0, 0xef00010100040200010004030000000000000160005000000000000000000000) return(0, 23) }'
+      - ':label deploying_valid_eof1 :yul { mstore(0, 0xef00010100040200010004040000000000000160005000000000000000000000) return(0, 23) }'
       # deployed code with data section
       # deployed code:
       #   code section: PUSH1(0) POP STOP 
       #   data section: aabbccdd
-      - ':label deploying_valid_eof1_with_data :yul { mstore(0, 0xef00010100040200010004030004000000000160005000aabbccdd0000000000) return(0, 27) }'
+      - ':label deploying_valid_eof1_with_data :yul { mstore(0, 0xef00010100040200010004040004000000000160005000aabbccdd0000000000) return(0, 27) }'
       # EOF initcode => EOF code
       # initcode without data section, deployed code without data section
-      # initcode code section: :yul { mstore(0, 0xef00010100040200010004030000000000000160005000000000000000000000) return(0, 23) }
       # deployed code section: PUSH1(0) POP STOP
-      - ':label eof1_initcode_deploying_valid_eof1 :raw 0xef0001010004020001002903000000000000027fef0001010004020001000403000000000000016000500000000000000000000060005260176000f3'
+      - ':label eof1_initcode_deploying_valid_eof1 :yul-eof { mstore(0, 0xef00010100040200010004040000000000000160005000000000000000000000) return(0, 23) }'
       # initcode without data section, deployed code with data section
-      # initcode code section: :yul { mstore(0, 0xef00010100040200010004030004000000000160005000aabbccdd0000000000) return(0, 27) }
       # deployed code:
       #   code section: PUSH1(0) POP STOP
       #   data section: aabbccdd
-      - ':label eof1_initcode_deploying_valid_eof1_with_data :raw 0xef0001010004020001002903000000000000027fef00010100040200010004030004000000000160005000aabbccdd0000000000600052601b6000f3'
+      - ':label eof1_initcode_deploying_valid_eof1_with_data :yul-eof { mstore(0, 0xef00010100040200010004040004000000000160005000aabbccdd0000000000) return(0, 27) }'
```

Comment:
there is now an argument support you can pass to the script like if you have custom solc version build with prague support (solc has --evm-version flag) 
so you can use it here without custom .sh as  `:yul prague {`

Reply:
Thanks for the heads up, Does this new feature still requires having a `yul-eof.sh` script or it is no longer needed?

Currently, as these changes in spec are recent, I'm using a `yul-eof.sh` script that converts the yul code.